### PR TITLE
fix: Improve copy/paste experience

### PR DESCRIPTION
**Pull request by armenzg**
It helps when copy pasting into JIRA.

### CODE REVIEW

File: text/0027-manual-disabling-of-flaky-tests.md [hunk 1]
```diff
@@ -28,25 +28,23 @@ https://sentry.io/organizations/sentry/dashboard/7997/?project=5350637&project=4
...
-          the following test is flaky so it is being disabled:
+```
+The following test is flaky so it is being disabled: <name of tests>
 
-          here is the sentry issue: <<< link here >>>
+Here is the sentry issue: <<< link here >>>
 
-          ```
-          <<< test failure output here >>>
-          ```
```
Comment (asottile-sentry) [inline text/0027-manual-disabling-of-flaky-tests.md:46]: I think a different trick that can be used here is to alternate backticks and ~ for the code fences -- I believe github supports both so it'd use the ~ on the outer fence and backticks on the inner fence?

File: text/0027-manual-disabling-of-flaky-tests.md [hunk 2]
```diff
@@ -28,25 +28,23 @@ https://sentry.io/organizations/sentry/dashboard/7997/?project=5350637&project=4
 **a test should be disabled if it has flaked on the primary branch**
 
 - identify the owner of the test in question
-    - often `.github/CODEOWNERS` can help
+    - often [.github/CODEOWNERS](https://github.com/getsentry/sentry/blob/master/.github/CODEOWNERS) can help
```
Comment (asottile-sentry) [inline text/0027-manual-disabling-of-flaky-tests.md:31]: this procedure isn't specific to `sentry` (for example `getsentry` also)

**Comment by armenzg** [comment]
I've addressed your last comment.

**Review by asottile-sentry — APPROVED** [review]

**Comment by armenzg** [comment]
Happy New Year!

**Pull request merged**